pkg/network/common/egress_dns.go

```diff
@@ -116,39 +143,41 @@ func (e *EgressDNS) Sync() {
 	}
 }
 
-func (e *EgressDNS) GetMinQueryTime() (time.Time, ktypes.UID, string, bool) {
+func (e *EgressDNS) GetMinQueryTime() (time.Time, string, []PolicyUpdate, bool) {
```

Comment:
This function's name was pretty non-obvious before, and is now worse. Maybe `GetNextUpdate`?

Reply:
Will change the function name.